<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureTransferDockWidgetBase</class>
 <widget class="QDockWidget" name="FeatureTransferDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Feature Transfer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="Copy_Field_Box">
      <property name="title">
       <string>Field(s) to Copy</string>
      </property>
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="Copy_Field_Text">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select field(s) to copy:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QListWidget" name="Field_Selection">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="Copy_Checkbox">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Copy All Fields</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="Copy_Feature_to_Layer_Button">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Copy Feature to Layer</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="Close_Button">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="Layers_Selection_Box">
      <property name="title">
       <string>Layers Selection</string>
      </property>
      <layout class="QFormLayout" name="formLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="Copy_from_Layer_Text">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Copy from Layer:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsMapLayerComboBox" name="Copy_from_Layer">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Paste_to_Layer_Text">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Paste to  Layer:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsMapLayerComboBox" name="Paste_to_Layer">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="Search_Box">
      <property name="title">
       <string>Search Source Layer for Features</string>
      </property>
      <layout class="QFormLayout" name="formLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="Select_Field_Text">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Select Field:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsFieldComboBox" name="Select_Field">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Search_String_Text">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search String:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QgsFeaturePickerWidget" name="Search_String">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="Search_Button">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFeaturePickerWidget</class>
   <extends>QWidget</extends>
   <header>qgsfeaturepickerwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Copy_Feature_to_Layer_Button</sender>
   <signal>clicked()</signal>
   <receiver>FeatureTransferDockWidgetBase</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Copy_from_Layer</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>Select_Field</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Search_Button</sender>
   <signal>clicked()</signal>
   <receiver>FeatureTransferDockWidgetBase</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Close_Button</sender>
   <signal>clicked()</signal>
   <receiver>FeatureTransferDockWidgetBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>